Task1.txt

```diff
@@ -0,0 +1 @@
+Test
```

Comment:
Please rewrite to more logical text (at least Line 1 or something else)

Reply:
Fixed
Thank you for your comment

Reply:
Have you added a commit?

Reply:
Yes, i did